tendrl_admin.yml

```diff
@@ -0,0 +1,5 @@
+---
+- hosts: localhost
+  tasks:
+    - name: Change admin password to admin123
+      tendrl_user: conf_path="usm.ini" new_password="admin123"
```

Comment:
Moreover I would suggest to remove `/root/password` file in this playbook to indicate that the default random password is no longer used. I don't see any value in modifying this file as well. What does @dahorak think?